contrib/devtools/Makefile

```diff
@@ -99,7 +99,7 @@ proto-tools-stamp:
 	@go install github.com/regen-network/cosmos-proto/protoc-gen-gocosmos
 
 	@echo "Installing protoc-gen-grpc-gateway..."
-	@go install https://github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
+	@go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
```

Comment:
This is incorrect -- we need to explicitly install 1.14.7. Where this installs `master` (i.e. dev), which means you need to do `go get ...`.

Reply:
i.e use `go get` instead of `go install`?

Reply:
yes.